setup.py

```diff
@@ -67,7 +67,7 @@
         "cryptography>=1.3, <1.6",
         "cssutils>=1.0.1, <1.1",
         "Flask>=0.10.1, <0.12",
-        "h2>=2.4.1, <3",
+        "h2>=2.4.1, <2.5",
```

Comment:
I had test failures for 2.5 locally, which was released today. This is to have a clean CI build here. We can bump this again in a separate PR and let CI decide.